lib/templates/image.mustache

```diff
@@ -0,0 +1,14 @@
+<div class="res-image">
+	{{#title}}
+	<h4 class="res-title">{{title}}</h4>
+	{{/title}}
+	{{#caption}}
+	<div class="res-image-caption">{{caption}}</div>
```

Comment:
I think these classes (and those for credits, and in galleries, etc) should be `res-caption res-image-caption` instead of just `res-image-caption`; otherwise the `.res-caption` styles are never used (AFAICT)